Adds extra documentation for WEBOTS_EXTRA_PROJECT_PATH. Docs to use the feature from Webots ROS (#4406)
cyberbotics/webots: docs/guide/preferences.md

Multiple user folders can be added by separating them with `:` (or `;` on Windows).
They may contain multiple sub-folders, each one associated to one sub-project (which should respect the [Standard File Hierarchy of a Project](the-standard-file-hierarchy-of-a-project.md)).
Alternatively, the environment variable `WEBOTS_EXTRA_PROJECT_PATH` can be used.

> If the `Extra project path` parameter and the `WEBOTS_EXTRA_PROJECT_PATH` environment variable are set, then both will be considered, having extra priority the paths set in the `Preferences` setting.

Reviewer: The second part of this sentence sounds odd to me, may I suggest this formulation?
```suggestion
If the `Extra project path` parameter and the `WEBOTS_EXTRA_PROJECT_PATH` environment variable are set, then both will be considered, with the priority given to the paths set in the `Preferences` setting.
```

Author: Sounds good, I applied your suggestion.
